### ISSUE TITLE
Newsletter SMTP Password is visible in plaintext to anyone with access

### ISSUE THREAD

**Issue by loudambiance** [Tautulli/Tautulli-Issues#172]
**Version:**
V2.1.28
**Branch:**
master
**Commit hash:**
e949b1486e67bc58c636ad13adad83a1747c5574

**Operating system:**
Linux 5.0.7-200.fc29.x86_64 (#1 SMP Mon Apr 8 15:40:59 UTC 2019 - Fedora 29 Twenty Nine)

**Python version:**
2.7.15 (default, Oct 15 2018, 15:26:09) [GCC 8.2.1 20180801 (Red Hat 8.2.1-2)]
**What you did?**
Inspect the "SMTP Password" text box in Newsletter settings

**What happened?**
Revealed email password in plain text

**What you expected?**
Did not expect to be able to view my email password.

**How can we reproduce your issue?**
\> Open Settings 
\> Newsletter Agents 
\> (Open an existing agent) 
\> Go to "Sending & Receiving Tab" 
\> Inspect the "SMTP Password" text box.
\> The HTML for the input box will have the value populated with the real password
![chrome_2019-04-16_20-29-22](https://user-images.githubusercontent.com/113403/56252616-8f71e100-6086-11e9-8169-da261c598c00.png)

**What are your (relevant) settings?**
Newsletter setup with SMTP.

**Link to logs:**
No logs.

**Comment by Arcanemagus** [comment]
So what you are saying is that the admin user can see what the admin user typed in. Please explain how this is an issue? Granted it shouldn't need to be sent out with the settings, but it's not a security issue.

Non-admin users can't see any of the settings so they don't have access to this in the first place.

**Comment by loudambiance** [comment]
In my specific case, it was a little complicated. I had just copied over config from a previous install, which I thought included the login/password, it did not have login enabled. This specific computer had pulled down an IPv6 address, so the entire world had could have viewed my email password. This is how I found out, when someone from outside my network accessed my install and changed all my users' names to inform me that my system was not secure. 

I do not know for a fact that they accessed my email. However, saying that "only and admin can see this" is not how security should be treated. I have already changed my email password, and secure Tautulli. I am just trying to prevent others from potentially falling into a similar situation. 

Further more, there are browser plugins, cross-site scripts, etc, specifically designed to try and capture data from fields like that.
